subsys/bluetooth/host/direction.c

```diff
@@ -7,15 +7,113 @@
 #include <assert.h>
 
 #include <bluetooth/hci.h>
+#include <bluetooth/l2cap.h>
 #include <bluetooth/conn.h>
+#include <bluetooth/direction.h>
 #include <sys/byteorder.h>
 
+#include "hci_core.h"
 #include "conn_internal.h"
+#include "direction_internal.h"
 
 #define BT_DBG_ENABLED IS_ENABLED(CONFIG_BT_DEBUG_HCI_CORE)
```

Comment:
Consider creating your own DEBUG statement, HCI_CORE enables a lot

Reply:
I remove it now because it is not required now. I will add separate DF debug entry in Kconfig when  such requirement arrives.

Reply:
Why is it not required? Without this line it will be enabled always due to this:
```
/* common/log.h */
#if !defined(BT_DBG_ENABLED)
#define BT_DBG_ENABLED 1
#endif
```

Reply:
Sorry, I didn't spot that. I'll fix it.

Reply:
Code is corrected and new Kconfig `CONFIG_BT_DEBUG_DF` added.